about/faq.rst

```diff
@@ -206,6 +206,12 @@ You can find detailed information on supported formats, how to export them from
 your 3D modeling software, and how to import them for Godot in the
 :ref:`doc_importing_3d_scenes` documentation.
 
+What renderer should I use?
+---------------------------
+
+To choose Renderer when making a new project, see: :ref:`doc_internal_rendering_architecture`
```

Comment:
You should check the [latest version of this page](https://docs.godotengine.org/en/latest/about/faq.html#should-i-use-the-forward-mobile-or-compatibility-renderer) 🙂